lib/ansible/modules/cloud/misc/virt_net.py

```diff
@@ -417,19 +417,24 @@ def set_autostart(self, entryid, state):
         return self.conn.set_autostart(entryid, state)
 
     def create(self, entryid):
-        return self.conn.create(entryid)
+        try:
+            return self.conn.create(entryid)
+        except libvirt.libvirtError as e:
+            if e.get_error_code() == libvirt.VIR_ERR_NETWORK_EXIST:
+                return None
```

Comment:
Did you intend to raise the original exception here when the condition is not met?

Reply:
hum, not really. I pushed a patch to fix that.